diff --git a/next.config.js b/next.config.js
index 1c3f056..07b37b4 100644
--- a/next.config.js
+++ b/next.config.js
@@ -2,6 +2,10 @@
 
 module.exports = {
   reactStrictMode: true,
+  trailingSlash: true,
+  skipTrailingSlashRedirect: true,
+  output: 'standalone',
+  assetPrefix: './',
   async rewrites() {
     return [
       {
diff --git a/src/components/Header.tsx b/src/components/Header.tsx
index 60a1696..61c631f 100644
--- a/src/components/Header.tsx
+++ b/src/components/Header.tsx
@@ -25,38 +25,8 @@ const Header = ({ page }: Props) => {
       <div className="flex items-center justify-center h-20">
         <div className="flex-1 inline-flex items-baseline">
           <Logo />
-          {isPlaygroundPage && (
-            <span className="text-xs ml-1 relative bottom-1 text-neutral-500 sm:flex hidden">
-              v{packageInfo.version}
-            </span>
-          )}
         </div>
         {isLearnDetail && <div id="ProgressArea" className="flex justify-center flex-1" />}
-        <div className="flex flex-1 items-center text-sm justify-end gap-2 sm:gap-4">
-          <IntlLink className="hidden md:block" navLink href="/[lang]/learn">
-            <FormattedMessage id="general.learn" />
-          </IntlLink>
-          <IntlLink className="hidden md:block" navLink href="/[lang]/cheatsheet">
-            <FormattedMessage id="general.cheatsheet" />
-          </IntlLink>
-          <IntlLink className="hidden md:block" navLink href="/[lang]/playground">
-            <FormattedMessage id="general.playground" />
-          </IntlLink>
-
-          <a
-            href="https://github.com/aykutkardas/regexlearn.com"
-            target="_blank"
-            rel="noreferrer"
-            aria-label="GitHub"
-            className={cx(
-              'text-neutral-200 hover:text-regreen-400',
-              'select-none relative items-baseline inline-flex justify-center',
-            )}
-          >
-            <Icon icon="github" size={18} />
-          </a>
-          <LanguageSelect />
-        </div>
       </div>
     </header>
   );
diff --git a/src/components/InteractiveArea.tsx b/src/components/InteractiveArea.tsx
index 67dd1df..59cfb34 100644
--- a/src/components/InteractiveArea.tsx
+++ b/src/components/InteractiveArea.tsx
@@ -231,7 +231,6 @@ const InteractiveArea = ({ isShow, setIsOpenModal }: Props) => {
         )}
         data-title={formatMessage({ id: 'general.regex' })}
       >
-        <ReportStep title={data.title} step={step} />
 
         {!data.noHint && (
           <Hint hiddenFlags={data.hiddenFlags} regex={data.regex} flags={data.flags} />
diff --git a/src/components/LessonBox.tsx b/src/components/LessonBox.tsx
index bf1b55a..d97e801 100644
--- a/src/components/LessonBox.tsx
+++ b/src/components/LessonBox.tsx
@@ -50,7 +50,7 @@ const LessonBox = ({ data, lock, bgColor }: Props) => {
     <DynamicWrapper className="hover:outline-8">
       <div
         className={cx(
-          'bg-[url(/images/noise.png)] relative bg-repeat bg-contain  transition-all duration-300 w-full h-44 bg-center rounded-xl py-3 px-4 flex flex-col shadow-xl hover:shadow-2xl flex-1 select-none',
+          'relative bg-repeat bg-contain  transition-all duration-300 w-full h-44 bg-center rounded-xl py-3 px-4 flex flex-col shadow-xl hover:shadow-2xl flex-1 select-none',
           bgColor || 'bg-[#324A34]/80 hover:bg-[#324A34]',
           !lock ? '' : 'cursor-not-allowed text-center grayscale',
         )}
diff --git a/src/components/Logo.tsx b/src/components/Logo.tsx
index 0c638b4..c425fc7 100644
--- a/src/components/Logo.tsx
+++ b/src/components/Logo.tsx
@@ -2,15 +2,14 @@ import Icon from 'src/components/Icon';
 import IntlLink from 'src/components/IntlLink';
 
 const Logo = () => (
-  <IntlLink href="/[lang]" className="inline-flex items-center" aria-label="RegexLearn">
-    <Icon
+  <Icon
       className="fill-regreen-400 block sm:hidden"
       icon="unlocked"
       removeInlineStyle
       size={20}
-    />
-    <img className="hidden sm:block" width={109} height={27} src="/logo.svg" alt="RegexLearn" />
-  </IntlLink>
+    >
+    <img className="hidden sm:block" width={109} height={27} src="./logo.svg" alt="RegexLearn" />
+  </Icon>
 );
 
 export default Logo;
diff --git a/src/context/InteractiveAreaContext.tsx b/src/context/InteractiveAreaContext.tsx
index f1f2d1b..3107b2b 100644
--- a/src/context/InteractiveAreaContext.tsx
+++ b/src/context/InteractiveAreaContext.tsx
@@ -63,6 +63,9 @@ const InteractiveAreaProvider = ({ lesson, lessonData, children }) => {
     setStep(currentStep);
     setLastStep(lastStep);
     setSuccess(currentStep < lastStep || data.readOnly || data.interactive === false);
+
+    const lsevent = new Event('lsupdated');
+    document.dispatchEvent(lsevent);
   }, [lookieKey, data]);
 
   const updateStorage = currentStep => {
diff --git a/src/data/lessons/index.json b/src/data/lessons/index.json
index c688a81..7ac36f8 100644
--- a/src/data/lessons/index.json
+++ b/src/data/lessons/index.json
@@ -6,19 +6,8 @@
     "description": "lessons.regex101.description",
     "creator": "Interactively",
     "creatorURL": "https://interactively.info/?utm_source=regexlearn&utm_medium=referral",
-    "creatorLogo": "/interactively.webp",
+    "creatorLogo": "./interactively.webp",
     "logoHeight": 18,
     "stepCount": 56
-  },
-  {
-    "key": "regexForSeo",
-    "slug": "regex-for-seo",
-    "title": "lessons.regexForSeo.title",
-    "description": "lessons.regexForSeo.description",
-    "sponsor": "ahrefs",
-    "sponsorURL": "https://ahrefs.com/?utm_source=regexlearn&utm_medium=referral&utm_campaign=sponsorship",
-    "sponsorLogo": "/ahrefs.webp",
-    "stepCount": 17,
-    "videoCount": 9
   }
 ]
diff --git a/src/data/lessons/regex101.js b/src/data/lessons/regex101.js
index d2943e7..78d8472 100644
--- a/src/data/lessons/regex101.js
+++ b/src/data/lessons/regex101.js
@@ -99,7 +99,7 @@ const regex101 = [
   {
     title: 'examples.starter.title',
     description: 'examples.starter.description',
-    image: '/Practise Starter.webp',
+    image: './Practise Starter.webp',
     interactive: false,
   },
   {
@@ -172,7 +172,7 @@ const regex101 = [
   {
     title: 'examples.finish.title',
     description: 'examples.finish.description',
-    image: '/Practise Starter.webp',
+    image: './Practise Starter.webp',
     interactive: false,
   },
   {
@@ -256,7 +256,7 @@ const regex101 = [
   {
     title: 'examples.starter.title',
     description: 'examples.starter.description',
-    image: '/Practise Starter.webp',
+    image: './Practise Starter.webp',
     interactive: false,
   },
   {
@@ -328,7 +328,7 @@ const regex101 = [
   {
     title: 'examples.finish.title',
     description: 'examples.finish.description',
-    image: '/Practise Starter.webp',
+    image: './Practise Starter.webp',
     interactive: false,
   },
   {
@@ -601,7 +601,7 @@ const regex101 = [
   {
     title: 'steps.completeAllSteps.title',
     description: 'steps.completeAllSteps.description',
-    image: '/Learn.webp',
+    image: './Learn.webp',
     interactive: false,
   },
 ];
diff --git a/src/localization/index.ts b/src/localization/index.ts
index c35de6a..ca04170 100644
--- a/src/localization/index.ts
+++ b/src/localization/index.ts
@@ -1,13 +1,5 @@
 export const langNames = {
   en: '🇺🇸',
-  de: '🇩🇪',
-  es: '🇪🇸',
-  fr: '🇫🇷',
-  tr: '🇹🇷',
-  ru: '🇷🇺',
-  'zh-cn': '🇨🇳',
-  uk: '🇺🇦',
-  ko: '🇰🇷',
   pl: '🇵🇱',
 };
 
diff --git a/src/pages/[lang]/learn/[lesson].tsx b/src/pages/[lang]/learn/[lesson].tsx
index 6b6f22f..566b197 100644
--- a/src/pages/[lang]/learn/[lesson].tsx
+++ b/src/pages/[lang]/learn/[lesson].tsx
@@ -21,7 +21,7 @@ const PageLesson = ({ lesson }: PageLessonProps) => {
   return (
     <>
       <Head>
-        <link rel="stylesheet" href="/css/animate.css" />
+        <link rel="stylesheet" href="./css/animate.css" />
       </Head>
       <InteractiveAreaProvider key={lessonData} lesson={lesson} lessonData={lessonData}>
         <div className="px-3 flex flex-col flex-1 justify-between relative overflow-x-hidden">
diff --git a/src/pages/[lang]/learn/index.tsx b/src/pages/[lang]/learn/index.tsx
index 00e1054..64c2e77 100644
--- a/src/pages/[lang]/learn/index.tsx
+++ b/src/pages/[lang]/learn/index.tsx
@@ -41,21 +41,20 @@ const PageLearn = () => {
                 <a
                   className="text-xs flex items-center justify-end text-neutral-400 hover:text-neutral-100 relative ml-auto mt-2 mr-2"
                   href={
-                    lesson.sponsorURL ||
                     lesson.creatorURL ||
                     'https://github.com/aykutkardas/regexlearn.com#sponsoring'
                   }
                   target="_blank"
                   rel="noreferrer"
                 >
-                  {lesson.sponsor || lesson.creator ? (
+                  {lesson.creator ? (
                     <span className="flex items-center">
-                      {lesson.sponsor ? 'Sponsored' : 'Created'} by{' '}
+                      {'Created'} by{' '}
                       <img
                         className="mx-1"
                         style={{ height: lesson.logoHeight || 12 }}
-                        src={lesson.sponsorLogo || lesson.creatorLogo}
-                        alt={lesson.sponsor || lesson.creator}
+                        src={lesson.creatorLogo}
+                        alt={lesson.creator}
                       />
                     </span>
                   ) : (
diff --git a/src/pages/_document.tsx b/src/pages/_document.tsx
index 59a585d..81053f9 100644
--- a/src/pages/_document.tsx
+++ b/src/pages/_document.tsx
@@ -1,6 +1,6 @@
 import Document, { Html, Head, Main, NextScript } from 'next/document';
 
-const isDevelopment = process.env.NODE_ENV === 'development';
+const isDevelopment = true;
 
 class MyDocument extends Document {
   render() {
@@ -29,7 +29,7 @@ j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
             rel="stylesheet"
           />
 
-          <link rel="shortcut icon" href="/favicon.svg" />
+          <link rel="shortcut icon" href="./favicon.svg" />
         </Head>
         <body>
           <Main />
diff --git a/src/utils/getIntlPath.ts b/src/utils/getIntlPath.ts
index 569593e..b8971bf 100644
--- a/src/utils/getIntlPath.ts
+++ b/src/utils/getIntlPath.ts
@@ -10,9 +10,9 @@ type GetIntlPathArgs = {
 const getIntlPath = ({ href, lang, query = {} }: GetIntlPathArgs): string => {
   let pathname = href;
 
-  if (lang === defaultLocale) {
-    pathname = pathname.replace('/[lang]', `/`).replace(/\/\//g, '/');
-  }
+  //if (lang === defaultLocale) {
+  //  pathname = pathname.replace('/[lang]', `/`).replace(/\/\//g, '/');
+  //}
 
   pathname = pathname.replace('/[lang]', `/${lang}`);
 
